tests/unit/db/redshift/test_unloader.py

```diff
@@ -33,3 +33,48 @@ def test_can_unload_this_format_true(self,
                                mock_unload_plan.records_format,
                                mock_processing_instructions.fail_if_cant_handle_hint)
         self.assertEqual(True, out)
+
+    def test_can_unload_to_scheme_s3_true(self):
+        mock_db = Mock(name='db')
+        mock_table = Mock(name='table')
+
+        redshift_unloader =\
+            RedshiftUnloader(db=mock_db,
+                             table=mock_table,
+                             s3_temp_base_loc=None)
+        self.assertTrue(redshift_unloader.can_unload_to_scheme('s3'))
+
+    def test_can_unload_to_scheme_file_with_temp_bucket_True(self):
```

Comment:
the most pedantic comment, but we're here.... True is capitalized in the function name but not in others.